File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/remote/response/SimilarResponse.java
package com.ryanrvldo.moviecatalogue.data.source.remote.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.ryanrvldo.moviecatalogue.data.source.model.Similar;

import java.util.List;

import lombok.Data;

@Data
public class SimilarResponse {
    @SerializedName("page")
    @Expose
    private int page;

    @SerializedName("results")
    @Expose
    private List<Similar> similar;

    @SerializedName("total_pages")
    @Expose
    private int totalPages;

    @SerializedName("total_results")
    @Expose
    private int totalResults;
}

File: app/src/main/kotlin/com/ryanrvldo/moviecatalogue/data/remote/response/SimilarResponse.kt
package com.ryanrvldo.moviecatalogue.data.remote.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.ryanrvldo.moviecatalogue.data.model.Similar

data class SimilarResponse(
    @Expose @SerializedName("page") val page: Int,
    @Expose @SerializedName("results") val similar: List<Similar>,
    @Expose @SerializedName("total_pages") val totalPages: Int,
    @Expose @SerializedName("total_results") val totalResults: Int

)